common/doc.go

```diff
@@ -1,3 +1,28 @@
 // Package common contains wrapper types and utilities common to all of the
 // OOXML document formats.
 package common
+
+import (
+	"archive/zip"
+	"strings"
+
+	"github.com/unidoc/unioffice"
+	"github.com/unidoc/unioffice/zippkg"
+)
+
+//AddImageToZip adds an image (either from bytes or from disk) and adds it to the zip file.
+func AddImageToZip(z *zip.Writer, img ImageRef, imageNum int, dt unioffice.DocType) error {
```

Comment:
Please, move this func to `zipping/helpers.go` - there are relative functions there. Also `doc.go` is a conventional name for an empty .go file containing only good lines for the whole package.

Reply:
Thanks for the clarification. I've created a new `zipping` package and put the function there.

Reply:
@mec07 I'm really sorry, my bad, I mistyped. I meant, please move this func to `zippkg/helpers.go`. There is already such file with relevant functions. It's not a mistype buy the way, Safari just keeps changing `zippkg` to `zipping`, sorry, didn't notice

Reply:
Hi @nkryuchkov, sorry for the slow response. There is a cyclical import issue when you put that function in `zippkg`, i.e. `common` imports `zippkg` and then we're trying to make `zippkg` import `common`... I've just put that function into `common/helpers.go`. Hopefully that is better.